### PR TITLE
Scrum 24 create home screen: Finished Welcome Back View

### PR DESCRIPTION
**Pull request by Dao-Ho**
## Description

[[Link to Ticket](insert the link to your JIRA ticket inside the parenthesis here)](https://www.figma.com/design/n4OxYALqlgaK7kxlnq7ilu/3-Stones?node-id=1392-4801&node-type=frame&t=bv7S10Oecp1ZO92v-0)

Added welcome back block to home page

## Screenshots:
welcome back block finished

### CODE REVIEW

**Comment by Dao-Ho** [comment]
Update: More accurate notification icon size
welcome back block finished

File: frontend/src/screens/home/home.tsx [hunk 1]
```diff
@@ -13,10 +15,104 @@ interface HomeScreenProps {
 
 const StyledView = styled(View);
 const StyledText = styled(Text);
+const StyledImage = styled(Image);
+
+const WelcomeBlock = ({ name }: { name: string }) => {
```
Comment (rmsap) [inline frontend/src/screens/home/home.tsx:20]: Maybe this was just here for testing purposes, but can we extract this WelcomeBlock to another file?

File: frontend/src/screens/home/home.tsx [hunk 2]
```diff
@@ -13,10 +15,104 @@ interface HomeScreenProps {
...
+};
+
+const PortfolioValue = ({
+  Portfoliovalue,
+  Portfoliochange,
+  navigation,
+}: {
+  Portfoliovalue: number;
+  Portfoliochange: number;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (rmsap) [inline frontend/src/screens/home/home.tsx:48]: Put a comment explaining why you disabled this rule (I'm guessing that the NavigationScreenProp takes in any, any so you didn't have a choice, but good to leave a comment saying that so its documented and future engineers are aware of that).

File: frontend/src/screens/home/home.tsx [hunk 3]
```diff
@@ -13,10 +15,104 @@ interface HomeScreenProps {
...
+              className='w-[7vw] h-[7vw]'
+            ></StyledImage>
+          </StyledView>
+        </StyledView>
+      </StyledView>
+    </StyledView>
+  );
+};
+
+const PortfolioValue = ({
```
Comment (rmsap) [inline frontend/src/screens/home/home.tsx:41]: Like above, if we can extract this to another file I think that would be great.

**Review by rmsap — CHANGES_REQUESTED** [review]
Generally good, a few couple comments:

1. Look for places in your TSX where you can extract out to make smaller components. In general seeing something like
```
<View>
     <View>
          <View>
               ...
          </View>
     </View>
</View>
```
is a sign that we may be able to extract out components so that it becomes 
```
<View>
     <View>
          <SomeComponent>
     </View>
</View>
```
or 
<View>
     <SomeComponent>
</View>
This is important because it makes code much easier to read, and consequently much more maintainable in the long run.
2. Can the wide white button and wide green button be combined into a single component that takes in a color prop? Perhaps I missed something but they looked basically identical to me besides the fact that they are a different color, so if we could just have a WideButton prop and then pass in the color that might make the code easier to maintain in the long run.

**Review by rmsap — APPROVED** [review]
Great job!

**Pull request merged**